PWGMM/Tasks/uecharged.cxx

```diff
@@ -332,5 +311,5 @@ struct uecharged {
 WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
 {
   return WorkflowSpec{
-    adaptAnalysisTask<uecharged>(cfgc, TaskName{"ue-charged"})};
+    adaptAnalysisTask<uecharged>(cfgc, TaskName{"UeCharged"})};
```

Comment:
Rather than use `TaskName` argument you can rename the struct itself instead, then you do not need `TaskName` at all.

Reply:
Now I followed what was done in some tasks from HF